### PR TITLE
DTS 2159378: Fix AggregateError--Part 4

### PR DESCRIPTION
**Pull request by garun-kumar**
**Task name**: DockerV0, DockerV1, HelmDeployV0, KubernetesManifestV1, KubernetesV1

**Description**: DTS 2159378

**Documentation changes required:** Y

**Added unit tests:** N

**Attached related issue:** [DTS TASK 2159378](https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2159378)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/KubernetesV1/_buildConfigs/Node20/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@types/q": "^1.5.0",
     "@types/uuid": "^8.3.0",
     "agent-base": "^6.0.2",
-    "azure-pipelines-task-lib": "^4.13.0",
+    "azure-pipelines-task-lib": "^4.11.0",
```
Comment (onetocny) [inline Tasks/KubernetesV1/_buildConfigs/Node20/package.json:10]: Why are we downgrading to older version?

File: Tests/task_outputs/build_1718601022170.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+successfully build test/test:2 image
```
Comment (onetocny) [inline Tests/task_outputs/build_1718601022170.txt:1]: What are we using these output files for? Do we need to have them in source control?

**Review by onetocny — COMMENTED** [review]
Thanks for PR. Please take a look at my questions.

File: Tests/task_outputs/build_1718601022170.txt (hunk 2 above)
Comment (starkmsu) [inline Tests/task_outputs/build_1718601022170.txt:1]: Please remove all new txt files from Tests/task_outputs. And please add exclusion rule for those to .gitignore

File: Tasks/DockerV0/task.loc.json [hunk 3]
```diff
@@ -341,10 +341,10 @@
   ],
   "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
   "execution": {
-    "Node10": {
```
Comment (starkmsu) [inline Tasks/DockerV0/task.loc.json:344]: Nothing changes - what's the purpose of this?

Comment (garun-kumar) [reply Tasks/DockerV0/task.loc.json:344]: I'm putting Node10 below Node16 as part of msal-node upgrade

File: Tests/task_outputs/build_1718601022170.txt (hunk 2 above)
Comment (garun-kumar) [reply Tests/task_outputs/build_1718601022170.txt:1]: These are auto generated files

Comment (garun-kumar) [reply Tests/task_outputs/build_1718601022170.txt]: Done

File: Tasks/KubernetesV1/_buildConfigs/Node20/package.json (hunk 1 above)
Comment (garun-kumar) [reply Tasks/KubernetesV1/_buildConfigs/Node20/package.json]: upgrade to 4.13.0. The reason I downgraded becauses I was changing it to 4.11.0 for all tasks. So didnt't notice that KubernetesV1 had 4.13.0. So reverted the change now.

**Review by onetocny — APPROVED** [review]

**Pull request merged**